src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -1825,18 +1825,18 @@ public int Count
     {
         get
         {
-            return ((IReadOnlyList<int>)field).Count;
+            return ((IReadOnlyCollection<int>)field).Count;
```

Comment:
these changes are desirable.  we're trying to get at members of base interfaces, so we only need to cast through that base interface.